src/containers/AdminCampaignEdit/sections/CampaignContactsForm/components/ConfigureColumnMappingDialog.tsx

```diff
@@ -225,8 +228,8 @@ const ConfigureColumnMappingDialog: React.FC<ConfigureColumnMappingDialogProps>
                           fullWidth
                           freeSolo
                           filterOptions={(
-                            options: string[],
-                            params: FilterOptionsState<string>
+                            options: unknown[],
```

Comment:
specifying `const filter = createFilterOptions<string>();
`
means you don't have to change these to unknown

Reply:
yupp this one i totally should've figured out lol